README.md

```diff
@@ -121,7 +121,6 @@ Unrecommended ones can still be considered. The most important thing is to remai
 ### E
 
 -  🟢 e • event
--  🟢 e.g. • example
```

Comment:
Shouldn't we add `ex`?

Reply:
I wasn't referencing "ex." because I thought would need to be in the list -- just because "example" wasn't the long form of "e.g." Quote from the [page](https://preply.com/en/question/example-ex-or-eg-41620) I linked:

> “Ex.” Has become a logical abbreviation or short form for the word example although its usage is not very common.

If this were to be included in the list, I would give it a red circle. But, in contrast to the existing entry "🔴 2 • to", I never saw it in identifiers or other formal elements.

Reply:
Ok, let's remove it for now, and we will later.